### DISCUSSION TITLE
Stuck situation with bumping fee opening channel

### DISCUSSION THREAD

**Discussion by Darth-Coin** [ZeusLN/zeus#2174] [Q&A] [answered]
I would like to hear other expert opinions in this matter. I was playing around with a new Zeus node and I end up in a stuck situation.
I will post it here, to remain documented and not bloat the Zeus telegram group.
Maybe this case could be useful to improve the user experience with opening channels and not get stuck.

*I don't care about showing real data, this is not my "production" Zeus node, is just a testing ground and could be reset anytime*

### Steps I did
1. I created a new Zeus node and sent the sats from another old wallet using Sparrow into a new unique utxo (my mistake, I should sent to 2 UTXOs).
2. From that UTXO in Zeus I opened a channel with Blixt node of 1M from that single UTXO of 1,65M (so 640k went to a change address). I choose fast confirmation in Zeus, but in mempool appeared with 14sats... idk why, when the highest fee was like 20sats/vB in mempool. 
3. So I tried to do an RBF from Zeus, but I got only a pending icon that never stopped. Practically it could do the RBF because the change UTXO was waiting the mempool and Zeus coouldn't use it anymore.
4. So I imported the Zeus keys into Sparrow and sent a new UTXO of 100k sats from another wallet, to be able to use RBF bump. So in total I had around 1,750,000 sats in that wallet.
5. I did the RBF in Sparrow with a good fee and the new tx got confirmed, but channel seems to not open.
6. Now in Sparrow all shows ok, all txs confirmed. But in Zeus is still pending.

### Transactions / outputs / device / enviornment
- My Zeus nodeID is 028d4cc557a5ff9c2c489e0a494fe68f322ed5139b9b101adc1cdeb48308978a34
- Android 12 / Lineage 19, Motorola G9 plus
- Zeus v0.8.4 rc2
- original channel opening tx from Zeus is this:
https://mempool.space/tx/3c162b498c3c477cd023c96059bd4b632dce2428863082bb09948fb59d4c0955#vout=1
- and the RBFed one from Sparrow, already confirmed is this one:
https://mempool.space/tx/a780d3e24fcaa923d1e11148ce311f74d023e59681f3fa3ffd3681642b2b1ef1

### Status, how is displayed
- Channel status in Zeus
![channel_status](https://github.com/ZeusLN/zeus/assets/56997124/d570b653-d781-4195-b7a0-8b138749da54)
- Sparrow transactions, confirmed. As you can see the original change address is empty and is showing the new change address after the RBF.
![image](https://github.com/ZeusLN/zeus/assets/56997124/c22981c1-1e13-4724-a135-36214495ffea)
- This is displayed in Zeus the change address, but without confirmations, but still showing in "coins" selection.
![change-address](https://github.com/ZeusLN/zeus/assets/56997124/e27117fc-e93f-4b61-a2f5-83c9e1646012)
- Zeus main balance, showing erroneously a pending balance of 1,636,410 (when in fact it should be only 936k - the opening channel) and 741k already confirmed, in 2 UTXOs (one of 643k change and the additional one of 100k).
![zeus-balance](https://github.com/ZeusLN/zeus/assets/56997124/d3de4f83-f5c7-4bd6-89f1-ee96d5c10f97)
- Zeus is displaying that I am already connected to Blixt peer.
![blixt-peer](https://github.com/ZeusLN/zeus/assets/56997124/870a98ba-59e0-4b4b-bc31-7952fe0747b9)

### Steps I did trying to recover that channel
- I generated a bunch of 10-15 new addresses in Zeus, comparing them also with the ones in Sparrow, all the same.
- I did a rescan and some restarts in Zeus
- I deleted the neutrino files and initiated again the rescan. That was the moment when the additional 100k sats UTXO appeared also in Zeus
- I tried to connect the LN peer Blixt node from the opening channels screen, but is saying that is already connected, so that Blixt node is "seeing" me, is not a connection issue.
- Zeus node is synced to chain and synced to graph, no connection issues, I selected the best neutrino peers

Zeus seems to working fine, the only thing that cannot do is to rescan that new change address from RBF and the opening channel.
I don't know anymore what to do, how to recover this channel or bring it online.

**Comment by kaloudis** [comment]
Related to the incorrect pending balance: https://github.com/ZeusLN/zeus/issues/2167

**Comment by djkazic** [comment, marked as answer]
DO NOT RBF CHANNEL OPENS! USE CPFP

**Comment by djkazic** [comment]
Recommend closing this issue as this is expected behavior from LND

**Comment by Darth-Coin** [comment]
@djkazic Thanks for clarifying. Yes was my mistake to do that RBF. Maybe I clicked wrong and instead of CPFP I clicked RBF. The fees were going fast up and that's why I tried to catch up to confirm that opening channel.
I posted this case in the "Discussions" Q&A section not in real Zeus issues because I wanted to discuss with you guys a situation like this.
This was a crazy test of mine, trying to put myself in a user situation doing these, so we can have better documentation and be prepared with the answers. Are many users doing this, so I wanted to see exactly what is going on.
Also it could help to build better tools and UI for Zeus and Blixt.
